airbyte-server/src/main/java/io/airbyte/server/converters/ConfigFileArchiver.java

```diff
@@ -94,9 +157,9 @@ private Path buildConfigPath(ConfigSchema schemaType) {
         validateJson(config, schemaType);
         results.add(config);
       }
-      LOGGER.debug(String.format("Successful read of airbyte config %s", schemaType));
+      LOGGER.debug(String.format("Successful read of airbyte config %s from archive", schemaType));
     } else {
-      LOGGER.debug(String.format("Airbyte config %s was not found", schemaType));
+      throw new ConfigNotFoundException(schemaType, "any");
```

Comment:
i might just use IOException. `ConfigNotFoundException` is specific to the `ConfigPErsistence` interface. This is just reading from something on disk. Not a big deal either way.

Reply:
or file not found like you do in DatabaseArchiver.